recipe/meta.yaml

```diff
@@ -96,7 +89,6 @@ about:
 
```

Comment:
unless strickly necesary we probably might want to remove the html tags from the `description` section
https://github.com/AnacondaRecipes/daal4py-feedstock/blob/e9ca7665d460abd904cd1b30d41d4c1a262cab5e/recipe/meta.yaml#L78&L86

We need to add the `license_family`:
https://github.com/AnacondaRecipes/daal4py-feedstock/blob/e9ca7665d460abd904cd1b30d41d4c1a262cab5e/recipe/meta.yaml#L73&L77

Reply:
Done. Thanks!